cypress/utils/utils.ts

```diff
@@ -129,6 +129,24 @@ export function click(fieldId: string, isForced = true, log = false): void {
     cy.get(fieldId, { log, timeout: 60 * SEC }).click({ log, force: isForced });
 }
 
+export function clickWithFocus(fieldId: string, isForced = true, log = false): void {
+    if (!log) {
+        cy.log(`Click ${fieldId}`);
+    }
+    cy.get(fieldId, { log, timeout: 60 * SEC })
```

Comment:
60 secs of timeout sounds like too much to me for most of the cases, maybe this should be a parameter with a default value of maybe 10 seconds at much. 

Same for the other method

Wdyt?

Reply:
Changed to 30 sec